src/test/java/com/reckue/post/services/realizations/CommentServiceRealizationTest.java

```diff
@@ -0,0 +1,318 @@
+package com.reckue.post.services.realizations;
+
+import com.reckue.post.PostServiceApplicationTests;
+import com.reckue.post.exceptions.ModelAlreadyExistsException;
+import com.reckue.post.exceptions.ModelNotFoundException;
+import com.reckue.post.models.Comment;
+import com.reckue.post.repositories.CommentRepository;
+import com.reckue.post.utils.Generator;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * Class CommentServiceRealizationTest represents test for CommentService class.
+ *
+ * @author Artur Magomedov
+ */
+public class CommentServiceRealizationTest extends PostServiceApplicationTests {
+    private Comment comment;
+    private Comment comment2;
+    private Comment comment3;
+
+    @Mock
+    private CommentRepository commentRepository;
+
+    @InjectMocks
+    private CommentServiceRealization commentService;
+
+    @BeforeEach
+    private void createComments(){
+        comment = Comment.builder()
+                .id("javascript")
+                .text("front")
+                .userId("vlad")
+                .postId("007")
+                .published(404L)
+                .build();
+
+        comment2 = Comment.builder()
+                .id("html")
+                .text("test")
+                .userId("ivery")
+                .postId("911")
+                .published(500L)
+                .build();
+
+        comment3 = Comment.builder().
+                id("java").
+                text("web").
+                userId("sherzod").
+                postId("666").
+                published(200L).
+                build();
+    }
+
+    @Test
+    public void create() {
+        String ID = Generator.id();
```

Comment:
Sorry, but i don't understand what the reason for generating id and creating the variable ID and not using it.

Reply:
Thanks for the careful review of the code! I will correct it in the next commit.